lib/shopify_api/resources/inventory_level.rb

```diff
@@ -0,0 +1,49 @@
+
+# frozen_string_literal: true
+module ShopifyAPI
+  class InventoryLevel < Base
+    def self.element_path(prefix_options = {}, query_options = nil)
+      prefix_options, query_options = split_options(prefix_options) if query_options.nil?
+      "#{prefix(prefix_options)}#{collection_name}.#{format.extension}#{query_string(query_options)}"
```

Comment:
Can you explain why do we need this?

Reply:
`element_path` is part of ActiveResource::Base and it helps construct the url for the custom method. The built in `[element_path](https://github.com/rails/activeresource/blob/b71ded7e1b9004c7a24f28203c35c7ad24da6617/lib/active_resource/base.rb#L769)` method is 

```ruby
def element_path(id, prefix_options = {}, query_options = nil)
  check_prefix_options(prefix_options)

  prefix_options, query_options = split_options(prefix_options) if query_options.nil?
  "#{prefix(prefix_options)}#{collection_name}/#{URI.parser.escape id.to_s}#{format_extension}#{query_string(query_options)}"
end
```

Which adds the id of resource in the path. So this becomes `/admin/inventory_levels/#{ inventory_level.id }.json?#{ params }`. We don't expose inventory_level ids so we need to send DELETE to the base `/admin/inventory_levels.json` endpoint with id params. Redefining element_path here removes the id and sends DELETE to the correct path. 

I was stuck trying to get this to go to the right URL, but our asset resource also needs a special path and I lifted the code from [asset.rb](https://github.com/Shopify/shopify_api/blob/master/lib/shopify_api/resources/asset.rb#L41).

In hindsight  though, I don't need to explicitly call it from destroy since its called by default.

Reply:
oh ok. What do you think about adding a comment with the same explanation you added here to this method? So just other devs can understand the need of it (and that is only needed become of the 'delete' call).

I saw that you were doing the same as in `asset.rb` but couldn't understand why.